be/src/exec/tablet_sink.cpp

```diff
@@ -128,55 +127,127 @@ Status NodeChannel::open_wait() {
     _open_closure = nullptr;
 
     // add batch closure
-    _add_batch_closure = new RefCountClosure<PTabletWriterAddBatchResult>();
-    _add_batch_closure->ref();
+    _add_batch_closure = ReusableClosure<PTabletWriterAddBatchResult>::create();
+    _add_batch_closure->addFailedHandler([&]() {
```

Comment:
I think it is better to declare all your capture other than a &. And what's more, referencing a local variable will lead to invalid memory access, because when the lambda is called, the reference is not valid.

Reply:
The two handlers both need to capture _rpc_error(the member variable), I'm afraid I can only change [&] to [this].
There's an alternate way to explicit capture, although I don't like this style. How about this?
```
auto& rpc_error_ref=_rpc_error;
_add_batch_closure->addFailedHandler([&rpc_error_ref](){})
```

Reply:
I think `[this]` is OK